### ISSUE TITLE
AccessibilityModulesType not exported + import documentation error

### ISSUE THREAD

**Issue by iquito** [ranbuch/accessibility#81]
I encountered some small issues that could easily be addressed:

- `AccessibilityModulesType` is not made available when importing the library, but it is mentioned in the documentation in the section `change modules order`, so I think this might just be an oversight
- The documentation on https://ranbuch.github.io/accessibility/site/ still says to import the library with `import { Accessibility } from 'accessibility/dist/main';`, which does not work - it should be `import { Accessibility } from 'accessibility';`, which is correct on Github, but not on the documentation page.

Otherwise thanks for the great package!

**Comment by iquito** [comment]
Another missing declaration: `interface IMenuInterface` is missing the `increaseLineHeight` and `decreaseLineHeight` methods, those are also missing in the documentation.

**Comment by iquito** [comment]
@ranbuch Yep all issues have been resolved, thanks!
